style.css

```diff
@@ -392,3 +369,97 @@ a {
   margin-bottom: 8px;
   margin-right: 8px;
 }
+
+.form {
+  background-image: url("assets/images/illustration-contact-form.svg");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+}
+
+.form h1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] For the form heading you are missing the font-family, Please include `font-family: Inter;` to your heading and make sure it matches the design.